cumulus_library_opioid/vocab/vsac.py

```diff
@@ -68,27 +69,26 @@ def download_oid_data(
     return True
 
 
-def main(cli_args=None):
-    """Temporary CLI interface"""
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "source_vocab", help="Name of vocab to look up codes for", default=None
-    )
-    parser.add_argument("--api_key", help="UMLS api key", default=None)
-    parser.add_argument(
-        "--force-recreate",
-        help="Force redownloading of data even if it already exists",
-        action="store_true",
-    )
-    parser.add_argument("--path", help="optional path to write data to", default=None)
-    args = parser.parse_args(cli_args)
-    return download_oid_data(
-        args.source_vocab,
-        api_key=args.api_key,
-        force_recreate=args.force_recreate,
-        path=pathlib.Path(args.path),
-    )
-
-
-if __name__ == "__main__":
-    main()
+def get_vsac_stewards(config:base_utils.StudyConfig) -> list[str]:
+    """Gets the steward target(s) from the StudyConfig options"""
+    stewards = []
+    try:
+        if config.options['steward'] == 'all':
```

Comment:
This will return KeyError, not TypeError if 'steward' isn't in the dict, yeah?

Reply:
ah yep, good catch, since this is the only "official" option i didn't catch that, will add a test as well